tests/test_server.py

```diff
@@ -181,6 +181,46 @@ async def test_autostart_batched(client, server, environment):
     assert len(sessionendpoint._sessions) == 1
 
 
+async def test_environment_setting_n_versions(client, server):
+    """
+    This test creates two environments with different values for the environment
+    setting AVAILABLE_VERSIONS_TO_KEEP and checks that the number of versions is
+    in line with the desired setting
+    """
+
+    env_1_expected_value = 30
```

Comment:
Why factor out these values into constants/variables if changing them will just break the testcase (i.e. the values are constants here, hardcoded elsewhere)